src/main/java/com/example/music/backend/playlist/converter/PlaylistResponseMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.example.music.backend.playlist.converter;
+
+import com.example.music.backend.playlist.domain.Playlist;
+import com.example.music.backend.playlist.response.PlaylistResponse;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+import java.util.List;
+
+@Mapper(componentModel = "string")
+public interface PlaylistResponseMapper {
+
+    PlaylistResponseMapper INSTANCE = Mappers.getMapper(PlaylistResponseMapper.class);
+
+    @Mapping(target = "ownerId", expression = "java(entity.getOwner().getId())")
```

Comment:
Look a bit weird. There is a better way to do the same thing?

Reply:
Replaced with source = "entity.owner.id", and still the same question, but why is it necessary?

Reply:
What do you mean with necessary?
https://stackoverflow.com/questions/56675587/when-is-expression-used-in-mapstruct